docs/03-architecture/supported-browsers.mdx

```diff
@@ -5,25 +5,18 @@ description: Browser support and which JavaScript features are supported by Next
 
 Next.js supports **modern browsers** with zero configuration.
 
-- Chrome 64+
-- Edge 79+
-- Firefox 67+
-- Opera 51+
-- Safari 12+
+- Chrome 111+
+- Edge 111+
+- Firefox 111+
+- Safari 16.4+
 
 ## Browserslist
 
 If you would like to target specific browsers or features, Next.js supports [Browserslist](https://browsersl.ist) configuration in your `package.json` file. Next.js uses the following Browserslist configuration by default:
 
 ```json filename="package.json"
 {
-  "browserslist": [
-    "chrome 64",
-    "edge 79",
-    "firefox 67",
-    "opera 51",
-    "safari 12"
-  ]
+  "browserslist": ["chrome 111", "edge 111", "firefox 111", "safari 16.4"]
```

Comment:
one question to this config in general. Based on browserlist website this config covers only specified versions. Not the range or >= 111.
See: 

<img width="1493" height="427" alt="Image" src="https://github.com/user-attachments/assets/bb014c73-be55-4b0a-86ae-2220a71cf5f9" />

Shouldn't this be something like:

<img width="1446" height="468" alt="Image" src="https://github.com/user-attachments/assets/e818dce3-d6d7-466a-8674-cf4d3f9a984b" />

As you state it's stable from certain browser version and "+".
Of course new coverage is still less than 50% but I'm not sure if you want to extend it further now.

Initially I wanted to create PR with the same question as I found it on Tuesday. Today I can see this PR so I will hold with it, maybe we can find answer here together

Reply:
yea, I want to believe that as well. I don't know why it itches me when I see it 😬 Why is this a thing in that case?

Anyway, does it bother you to have `>=` there? I can prepare PR if it's minor for you ✌️ 🙏